.github/workflows/build.yml

```diff
@@ -66,6 +70,17 @@ jobs:
             --api-version 6.5 --system-cxx \
             --target-dir build
           ninja -C src/out/build
+          
+      - name: Build for Tizen 8.0
```

Comment:
I warned the Tizen UI framework team about the backward incompatibility but they proceeded with the API removal without providing any alternative API.

We need to inform the users that apps built for previous versions of Tizen (e.g. Tizen 6.5) are not compatible with Tizen 8.0+ devices. They have to upload two copies of TPK to the app store to support both <8.0 and 8.0+. 

Alternatively we can simply remove the functionality (clipboard support) or use `dlopen`.

Reply:
I will remove the existing cbhm code and make a PR again using the ecore_wl2 API. https://github.com/flutter-tizen/embedder/pull/64
Perhaps I expect copy&paste to be supported in all versions and TV and common profiles.